source/manual/move-apps-between-servers.html.md

```diff
@@ -0,0 +1,103 @@
+
+1. Change the [hieradata] for the load balancers to balance between both the old and new servers. For this step, it is important that all servers are running the same version of the app, so if appropriate, re-deploy the app now.
+1. Deploy puppet in staging and production.
+
+The app will now be running on both old and new servers and the load balancers will use both as their configuration is updated by puppet.
+
+[hieradata]: https://github.com/alphagov/govuk-puppet/pull/7285
+
+### Remove the old servers from the load balancers
+
+Once all the load balancers have been updated, requests for the app(s) will be using both sets of servers, which will allow you to remove the old servers from serving the app(s).
+
+1. Change the [hieradata] once more to remove the old servers from the load balancers for the relevant app(s), and also to remove the app(s) from the old servers since they will no longer be serving that app.
+1. Deploy puppet in staging and production.
+1. Change the relevant [app deployment scripts][deploy-scripts] to deploy to only the new servers.
+
+> **Warning**
+> Bundling up the removal of the old servers from the load balancers and the removal of the app(s) from the old servers may result in a period where the old servers are still part of the load balancer group but don't have the app(s) running. This can be mitigated by either splitting up these changes, or running puppet manually on the load balancers after deployment to ensure no further traffic is routed to the old servers.
+
+[hieradata]: https://github.com/alphagov/govuk-puppet/pull/7310
+[deploy-scripts]: https://github.com/alphagov/govuk-app-deployment/pull/250
+
+### Clean up
+
+Once everything is done, make some final changes to the [puppet configuration and hieradata][puppet-changes] to clean up the temporary changes from above.
+
+[puppet-changes]: https://github.com/alphagov/govuk-puppet/pull/7311
+
+## AWS
+
+> **Note**
+> You need to be at least a Power User in AWS to be able to run the following procedure. You can check by looking in the [govuk-aws-data] repository.
+
+1. Add Terraform configuration ([1][aws-terraform-config-1], [2][aws-terraform-config-2], [3][aws-terraform-config-3]) to create the new servers, load balancers, security groups, DNS entries etc.
+1. Add data to complement the configuration above ([1][aws-terraform-data-1], [2][aws-terraform-data-2]).
+1. [Deploy][] the Terraform configuration. You need to do this three times for each environment:
+  1. `infra-security-groups` project in the `govuk` stack.
+  1. `app-email-alert-api` project in the `blue` stack.
+  1. `infra-public-services` project in the `govuk` stack (only if your app is accessible directly from the public internet).
+
+For each deployment, set the environment to one of `integration`, `staging` or `production` (only `integration` is currently live) and run the `plan` command first to double-check the changes before running the `apply` command to make the changes.
```

Comment:
may want to be careful about the "(only `integration` is currently live)" as that is quite time sensitive.

Reply:
I've removed that part.